docs/BasicCommunication/OnSystemRequest/index.md

```diff
@@ -66,7 +68,7 @@ BC.OnSystemRequest in "Proprietary" Policy Table Update Flow
 
 |||
 BC.OnSystemRequest in External Proprietary Policy Table Update Flow
-![EXTERNAL proprietary](../policyupdate/assets/diagram_PolicyUpdate_external_proprietary.png)
+![EXTERNAL proprietary](./assets/diagram_PolicyUpdate_external_proprietary.png)
```

Comment:
I would prefer if this diagram were referenced from the `PolicyUpdate` assets, rather than having a duplicate diagram.

Reply:
This diagram referenced is available in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/feature/read_generic_network_signal_data/docs/BasicCommunication/PolicyUpdate/index.md 
Removed from here

Reply:
It doesn't hurt to reference it here, but we shouldn't be maintaining two copies of the same diagram. On that note, you did not delete the duplicate diagram